draft-ietf-tls-tls13.md

```diff
@@ -1467,7 +1467,7 @@ ServerKeyShare
   Secret (in this mode they are the same).  [{{server-key-share}}]
 
 ServerConfiguration
-: supplies a configuration for a future handshake (see {{cached-server-configuration}}).
+: supplies a configuration for zero-rtt handshakes (see {{zero-rtt-exchange}}).
```

Comment:
For terminology consistency, I'd suggest using "0-RTT handshakes" instead of "zero-rtt handshakes" here. The former is used in most places.